execution-engine/src/wasi/common.rs

```diff
@@ -447,7 +923,7 @@ impl WasiWrapper {
     }
 
     /// The implementation of the WASI `args_sizes_get` function. It requires an extra `memory_ref` to
-    /// interact with the execution engine.
+    /// interact with the execution engine.                  
```

Comment:
There are a few occurrences of that extra whitespace

Reply:
Oh strange, these changed a lot more during development, so I think it was a mix of rustfmt changes on main and a mistake during rebasing. Can remove